agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -186,6 +187,14 @@ function install_k8s_agent {
     fi
 
     sed -i -e "s|# serviceAccount: sysdig-agent|serviceAccount: sysdig-agent|" /tmp/sysdig-agent-daemonset-v2.yaml
+    # add label for Sysdig instance
+    # -i.bak argument used for compatibility between mac (-i '') and linux (simply -i) 
+     if [ ! -z "$SYSDIG_INSTANCE_NAME" ]; then
```

Comment:
can you make sure the spacing on this aligned with the rest of the stanza? It feels like there is an extra space indent on this line.

Reply:
right - removed the extra space ...